package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "chalk": "2.4.1",
     "clui": "0.3.6",
-    "couch2pg": "^0.4.1",
+    "couch2pg-njuguna": "^0.5.0",
```

Comment:
Using this package to make it easier to test and not have to wait for the accompanying couch2pg change to me merged and released. It will be changed back to `couch2pg` and updated to the correct version number before merging.

Reply:
I've published `couch2pg@0.5.0` and reverted this change.